File: MeetingServer/MeetingSystem/Structure/VoteDataItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MeetingSystem
{
    public class VoteDataItem
    {
        private int id = -1;
        private List<VoteData> candidateList = new List<VoteData>();
        public int Id
        {
            get { return id; }
            set { id = value; }
        }
        public List<VoteData> CandidateList
        {
            get { return candidateList; }
            set { candidateList = value; }
        }
        public class VoteData
        {
            private String name;
            private Boolean agreeChoose;
            private Boolean rejectChoose;
            private Boolean invalidateChoose;

            public Boolean InvalidateChoose
            {
                get { return invalidateChoose; }
                set { invalidateChoose = value; }
            }
            public String Name
            {
                get { return name; }
                set { name = value; }
            }
            public Boolean AgreeChoose
            {
                get { return agreeChoose; }
                set { agreeChoose = value; }
            }
            public Boolean RejectChoose
            {
                get { return rejectChoose; }
                set { rejectChoose = value; }
            }
            public void print()
            {
                Console.Write(name + "/" + agreeChoose + "/" + rejectChoose + "/" + invalidateChoose);
            }
        }

        public void print()
        {
            Console.Write("id:" + Id);
            Console.Write("   candidate:");
            for(int i =0; i < CandidateList.Count(); i++)
            {
                CandidateList.ElementAt(i).print();
                Console.Write("   ");
            }
            Console.WriteLine("");
        }
    }
}

File: MeetingClient/MeetingSystem/app/src/main/java/com/ntut/structure/VoteDataItem.java
package com.ntut.structure;

import java.util.ArrayList;
import java.util.List;

import android.widget.ListAdapter;

public class VoteDataItem {
	private int id;
	List<voteData> candidateList = new ArrayList<voteData>();
	
    public class voteData
	{
        private String name;
        private Boolean agreeChoose;
        private Boolean rejectChoose;
        private Boolean invalidateChoose;
		public String getName() {
			return name;
		}
		public void setName(String name) {
			this.name = name;
		}
		public Boolean getAgreeChoose() {
			return agreeChoose;
		}
		public void setAgreeChoose(Boolean agreeChoose) {
			this.agreeChoose = agreeChoose;
		}
		public Boolean getRejectChoose() {
			return rejectChoose;
		}
		public void setRejectChoose(Boolean rejectChoose) {
			this.rejectChoose = rejectChoose;
		}
		public Boolean getInvalidateChoose() {
			return invalidateChoose;
		}
		public void setInvalidateChoose(Boolean invalidateChoose) {
			this.invalidateChoose = invalidateChoose;
		}	
	}

	public List<voteData> getCandidateList() {
		return candidateList;
	}

	public void setCandidateList(List<voteData> candidateList) {
		this.candidateList = candidateList;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}
}
